libsolidity/codegen/ArrayUtils.h

```diff
@@ -76,9 +72,9 @@ class ArrayUtils
 	/// Stack pre: reference
 	/// Stack post:
 	void popStorageArrayElement(ArrayType const& _type) const;
-	/// Appends a loop that clears a sequence of storage slots of the given type (excluding end).
-	/// Stack pre: end_ref start_ref
-	/// Stack post: end_ref
+	/// Appends a loop that clears a sequence of storage slots of the given type.
+	/// Stack pre: start_ref slotCount
```

Comment:
For whatever reason, we're using a different style for these. Let's stick to it for consistency.

```suggestion
	/// Stack pre: start_ref slot_count
```